staging/src/k8s.io/apiserver/pkg/server/filters/wrap.go

```diff
@@ -30,7 +30,7 @@ import (
 func WithPanicRecovery(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		defer runtime.HandleCrash(func(err interface{}) {
-			http.Error(w, "This request caused apisever to panic. Look in log for details.", http.StatusInternalServerError)
+			http.Error(w, "This request caused apiserver to panic. Look in log for details.", http.StatusInternalServerError)
```

Comment:
here it is `apiserver`, not `APIServer`. I tend to the former.

Reply:
Also: Look in the logs for details.

Reply:
ok